rubin_sim/maf/mafContrib/TDEsPopMetric.py

```diff
@@ -86,7 +85,10 @@ def __init__(
         self.ptsNeeded = ptsNeeded
 
         self.lightcurves = Tde_lc(file_list=file_list)
-        self.mjd0 = mjd0
+        if mjd0 is None:
+            self.mjd0 = survey_start_mjd()
+        else:
+            self.mjd0 = mjd0
```

Comment:
This is a classic case where a ternary operator might be good:
`self.mjd0 = survey_start_mjd() if mjd0 is None else mjd0`

Really, though, it doesn't matter.

Reply:
I like it!